mruby-mspec-support/mrblib/mruby-mspec-support.rb

```diff
@@ -27,10 +22,6 @@ def readlines(path)
       end
       a
     end
-
-    def split(path)
-      [File.dirname(path), File.basename(path)]
-    end
```

Comment:
I agree that we should get implemented in a mrbgems.

However, require-spec should remove dependency to File.split.
But I also think that it's difficult to remove all dependency.

It's OK like this.